nncf/common/graph/graph.py

```diff
@@ -71,6 +101,39 @@ class NNCFGraphNodeType:
     OUTPUT_NODE = MODEL_OUTPUT_OP_NAME
 
 
+class NNCFGraphEdge:
```

Comment:
Please add docstring for classes, methods here and below.

Reply:
Done